pnpm-lock.yaml

```diff
@@ -181,6 +181,16 @@ importers:
       '@angular/core': 15.1.5_rxjs@7.5.7+zone.js@0.11.8
       tslib: 2.5.0
 
+  libs/shared/components:
```

Comment:
is pnpm-workspaces file still exists? if so remove it, revert this file, remove node_modules, and run `pnpm i` again

Reply:
pull and resolve conflicts, it'll probably fix it